introduction.md

```diff
@@ -1,2 +1,6 @@
-+Introduction
-+My favorite Code School path is HTML & CSS
\ No newline at end of file
+Introduction
+My favorite Code School path is HTML & CSS
+
+* Git
+* JavaScript
+* Ruby
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?